src/lib/mongoBackend/mongoGetSubscriptions.cpp

```diff
@@ -173,7 +176,7 @@ void mongoListSubscriptions
   std::string                    conds = std::string(CSUB_CONDITIONS) + "." + CSUB_CONDITIONS_TYPE;
   BSONObj                        q     = BSON(conds << "ONCHANGE");
 
-  if (!collectionQuery(getSubscribeContextCollectionName(tenant), q, &cursor, &err))
+  if (!collectionRangedQuery(getSubscribeContextCollectionName(tenant), q, limit, offset, &cursor, &count, &err))
```

Comment:
For pagination, don't we need to order by creation date?
[ In case a new subscription is created between two paginated request, if the order of the subscriptions is altered, the pagination is ruined ... ]